tests/aggregation/tests.py

```diff
@@ -576,7 +576,7 @@ def test_distinct_on_stringagg(self):
         books = Book.objects.aggregate(
             ratings=StringAgg(Cast(F("rating"), CharField()), Value(","), distinct=True)
         )
-        self.assertEqual(books["ratings"], "3,4,4.5,5")
+        self.assertCountEqual(books["ratings"].split(","), ["3", "4", "4.5", "5"])
```

Comment:
This seemed more appropriate than passing `order_by="pk"` to the aggregate as that would have required gating the test with `supports_aggregate_order_by_clause`.